### PR TITLE
Add support to string l10n in Madara

### PR DESCRIPTION
**Pull request by alessandrojean**
Marking it as WIP since it may have a build error, since I didn't check all overrides.

Changelog:

- Add support for string localization in the base class;
- Hide non-manga titles (text and video) by default in the selector;
- Add additional manga title selector;
- Fix missing SS icons.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
Should be okay for review and merge now.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt [hunk 1]
```diff
@@ -222,42 +222,110 @@ abstract class Madara(
...
-            Pair("Latest", "latest"),
-            Pair("A-Z", "alphabet"),
-            Pair("Rating", "rating"),
-            Pair("Trending", "trending"),
-            Pair("Most Views", "views"),
-            Pair("New", "new-manga")
-        )
+    protected open val authorFilterTitle: String = when (lang) {
+        "pt-BR" -> "Autor"
+        else -> "Autor"
```
Comment (jopejoe1) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:227]: `else -> "Autor"` should be `else -> "Author"`

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt [hunk 2]
```diff
@@ -222,42 +222,110 @@ abstract class Madara(
...
+    }
+
+    protected open val artistFilterTitle: String = when (lang) {
+        "pt-BR" -> "Artista"
+        else -> "Artist"
+    }
+
+    protected open val yearFilterTitle: String = when (lang) {
+        "pt-BR" -> "Ano de lançamento"
+        else -> "Year of Released"
```
Comment (vetleledaal) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:237]: Year of Release?

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt [hunk 3]
```diff
@@ -222,42 +222,110 @@ abstract class Madara(
...
+    }
+
+    protected open val statusFilterTitle: String = when (lang) {
+        "pt-BR" -> "Estado"
+        else -> "Status"
+    }
+
+    protected open val statusFilterOptions: Array<String> = when (lang) {
+        "pt-BR" -> arrayOf("Completo", "Em andamento", "Cancelado", "Pausado")
+        else -> arrayOf("Completed", "Ongoing", "Canceled", "On Hold")
```
Comment (vetleledaal) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:247]: Cancelled. Also double check statusFilterOptionsValues

Comment (vetleledaal) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:247]: On Hold -> is title case correct here? related strings in main tachi do not use it.
```xml
<string name="unknown_status">Unknown status</string>
<string name="licensed">Licensed</string>
<string name="publishing_finished">Publishing finished</string>
<string name="cancelled">Cancelled</string>
<string name="on_hiatus">On hiatus</string>
```
Title case is used elsewhere, so it should be consistent if you change it here

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt (hunk 2 above)
Comment (jopejoe1) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:237]: This is following the default madara site.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt (hunk 3 above)
Comment (jopejoe1) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:247]: This is also following the defaul madara site

Comment (alessandrojean) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:247]: Just confirmed, it was like that before and the values are correct:

```kotlin
Tag("canceled", "Canceled"),
```

**Comment by CarlosEsco** [comment]
Has a merge conflict

**Comment by alessandrojean** [comment]
> Has a merge conflict

Fixed in the last two commits :+1:

**Pull request merged**